source/test/TestApp/PlayFabEventTest.cpp

```diff
@@ -169,38 +169,24 @@ namespace PlayFabUnit
         EmitEvents(PlayFab::PlayFabEventType::Heavyweight, maxBatchWaitTime, maxItemsInBatch, maxBatchesInFlight);
     }
 
-    /// EVENTS API
-    /// non-PlayStream lightweight events (emitted individually
-    ///   and processed in a background thread using event pipeline (router, batching, etc))
-    void PlayFabEventTest::LightweightEvents(TestContext& testContext)
-    {
-        eventTestContext = std::make_shared<TestContext*>(&testContext);
-
-        EmitEvents(PlayFab::PlayFabEventType::Lightweight);
-    }
-
-    void PlayFabEventTest::LambdaCallbackTest(TestContext& testContext)
-    {
+    /// OneDS lightweight events (emitted individually
+    void PlayFabEventTest::OneDSEventsApi(TestContext& testContext)
```

Comment:
>void PlayFabEventTest::OneDSEventsApi(TestContext& testContext) [](start = 4, length = 63)

Something is screwed up with the formatting here - where are the opening braces for this method ? Unless it's not showing up in codeflow

Reply:
Refresh. That was a merge conflict.